### PR TITLE
Refact/bud 322: backend for receipts filter api

### PR DESCRIPTION
**Pull request by shahdyousefak**
### BUD Link
[_BUD 322_](https://jira.budgetlens.tech/browse/BUD-322)

### Include api paths (if applicable)
api/receipts/filters/

### Summary of the PR
Adds a proper endpoint for receipt filter loading. Fixes a bug with loading receipt filters completely wrong before.

### Details
N/A

### Checks
- [x] Tested Changes
- [x] Ensured that changes do not affect other applications and its use (if applicable)

### CODE REVIEW

File: users/views.py [hunk 1]
```diff
@@ -117,11 +117,11 @@ def get(self, request, *args, **kwargs):
                 "last_name": user_profile.user.last_name,
                 "email": user_profile.user.email,
                 "telephone_number": str(user_profile.telephone_number),
-                "forwardingEmail": user_profile.forwarding_email
```
Comment (IsaacMuriuki) [inline users/views.py]: you probably need to change this back to `forwarding_email` after Justins fix in PR #86

**Review by ThatJustin — DISMISSED** [review]
Tested and works, thank you.

File: receipts/views.py [hunk 2]
```diff
@@ -189,3 +189,33 @@ def post(self, request, *args, **kwargs):
             return Response(receipt_serializer.data, status=status.HTTP_201_CREATED)
         else:
             return Response(receipt_serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+class ReceiptsAvailableFilters(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        print()
```
Comment (mamineahmadi) [inline receipts/views.py:198]: Useless print() statement?

**Review by mamineahmadi — DISMISSED** [review]

**Review by IsaacMuriuki — DISMISSED** [review]
lgtm

**Review by ThatJustin — APPROVED** [review]
All good again. Nice catch for the useless print.

**Review by IsaacMuriuki — APPROVED** [review]

**Review by teoPalomino — APPROVED** [review]
Thanks for this refactor!

**Pull request merged**